controllers/front/CmsController.php

```diff
@@ -224,4 +227,14 @@ public function getTemplateVarCategoryCms()
 
         return $categoryCms;
     }
+
+    public function getCms()
```

Comment:
is there a point when property is public?

Reply:
No, but I wanted to unify it, so we have one way of accessing this data by getters. And on the other side, I didn't want to break modules by setting the `$cms` as `protected`.